docs/pages/how-it-works/deploy-k8s.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: How it works using kubernetes
+sidebar_label: Deploying kubernetes
+---
+
+DevPod works the same with kubernetes as with Machines, the key difference is the secure tunnel is set up using the kubernetes control plane (e.g. kubectl ...) so an agent is not necessary
```

Comment:
`so an agent is not necessary` 
Technically we still have a local agent and one in the workspace container, just not one in the VM. What do you think of removing that part?

Reply:
this was the complete sentence - `so an agent is not necessary to be run on the kubernetes node`. I wanted to add this as looking at machine you could easily think the devpod agent is on the k8s node as well as the pod

Reply:
I can remove, but I think it's a good distinction

Reply:
@pascalbreuninger, @bkneis you can use `vale` to ensure `DevPod` is always spelled with proper capitalization. LMK if you are interested, and I can set it up for you.